submissions/Lu-sure/tiny-JS-world/index.js

```diff
@@ -0,0 +1,68 @@
+/* Refer to https://github.com/OleksiyRudenko/a-tiny-JS-world for the task details
+   Complete the below for code reviewers' convenience:
+
+   Code repository: https://github.com/Lu-sure/Lu-sure.github.io/tree/main/A_tiny_JS_world
+   Web app: https://lu-sure.github.io/A_tiny_JS_world/
+   */
+
+  const dog = {
+    species: 'dog',
+    name: 'Easy',
+    gender: 'female',
+    age: 2,
+    legs: 4,
+    hands: 0,
+    saying: 'woof-woof!',
+    friendly: 'everyone'
+  };
+ 
+  const cat = {
+    species: 'cat',
+    name: 'Roxy',
+    gender: 'female',
+    age: 1,
+    legs: 4,
+    hands: 0,
+    saying: 'meow!!',
+    friendly: 'Lila'
+  };
+ 
+  const man = {
+    species: 'human',
+    name: 'Kolia',
+    gender: 'male',
+    age: 26,
+    legs: 2,
+    hands: 2,
+    saying: 'Let`s work all day long!',
```

Comment:
Check [special characters escaping](https://www.w3schools.com/js/js_strings.asp) to avoid situations where you use a **similar** character (`` ` ``) in place of the proper one (`'`).
**You do not need to fix this** here, but make sure that you will address similar situations properly in your future submission.

Reply:
Yes, you are right. Was unobservant..